scripts/tests/testUnit.sh

```diff
@@ -4,4 +4,4 @@
 export NODE_ENV='test'
 
 echo 'Start firestore emulator and run unit tests:'
-firebase emulators:exec 'nyc --x=controllers --x=test --x=docs --x=mockdata mocha test/unit/**'
+firebase emulators:exec 'nyc --x=controllers --x=test --x=docs --x=mockdata mocha test/unit/**' --project='<your firestore app name>'
```

Comment:
No longer needed. Reason -

Reply:
ok!

Reply:
Please remove this as per shubham's comment as he has raised a PR that does the same